### PR TITLE
Remove all smee.io files

### LINKED ISSUE #87: Separate smee-client and smee-server into separate repositories
I'd like to chat about moving `smee-client` and the actual server into two separate repositories. My main motivation to do that is to ease contributions (contributing to the two is confusing, including for me) make build times faster (CI will have to test half the code).

What do y'all think? Is this something worth the small amount of effort, or is it _less_ confusing the way it is now?

cc @tcbyrd

### PR DESCRIPTION
**Pull request by JasonEtco**
This PR removes everything that is the `server` folder, and any CI things that were in place to make it work (see changes to the `.travis.yml` file). I've created the new repository [`probot/smee.io`](https://github.com/probot/smee.io) (private until its all setup, to avoid confusion), and will merge this once the following has been completed:

- [x] CI is up an running in `probot/smee.io`
- [x] We've deployed https://smee.io from that repo (and connected the Heroku pipeline)
- [x] ✅ on this PR

Closes #87 

It looks like CI is failing because the test suite for the client library is trying to require the server's entrypoint. Not sure how we'll solve that 🤔 We might choose to move that test over to the `smee.io` repo?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,41 @@
-# smee &middot; [![Build Status](https://img.shields.io/travis/probot/smee/master.svg)](https://travis-ci.org/probot/smee) [![Codecov](https://img.shields.io/codecov/c/github/probot/smee.svg)](https://codecov.io/gh/probot/smee/)
+# smee-client
```
Comment (tcbyrd) [inline README.md]: Should we add something at the top of the README here about moving the server bits to the other repo?

Comment (JasonEtco) [reply README.md]: 100% yep - was planning on doing that in a followup, but you're right, it should be part of this PR. Will add 👍

**Comment by JasonEtco** [comment]
Done @tcbyrd! Just need to resolve this failing test. I'm thinking of removing it, and moving it over to `probot/smee.io` and doing `require('smee-client')`, that way we still keep the interop. Does that sound reasonable to you?

**Comment by tcbyrd** [comment]
If I understand what you're saying (and maybe I don't), would that leave us with no tests in this repo? Should write some small unit tests for only the methods in the client?

**Comment by JasonEtco** [comment]
Because of the many many conflicts, I'm going to restart the other repo (delete it and make it again 😓). 

To answer your question @tcbyrd - we'd still have one test for `Client#createChannel`, but generally yeah there'd be way fewer tests here.

**Comment by JasonEtco** [comment]
Ok I think this is good to go @tcbyrd. I've [added an interop test](https://github.com/probot/smee.io/blob/master/tests/smee-client.test.js) in `probot/smee.io` (to be made public after this PR is approved, but before its been merged). I've removed all the `/server` code from this repo. We're definitely missing tests, but I really don't want to make this PR any larger than it needs to be 😅

**Comment by JasonEtco** [comment]
Thanks @tcbyrd! I've deployed `smee.io` from the new repo in Heroku and setup Travis. With that, I'll merge this PR 🎉

**Pull request merged**